Integrations/Ipstack/Ipstack_description.md

```diff
@@ -0,0 +1,98 @@
+## Overview
+---
+Use Ipstack to get location on IPs
+
+## Ipstack Playbook
+---
+
+## Use Cases
+---
+  * Get Location info about an IP
+## Configure Ipstack on Demisto
+---
+
+1. Navigate to __Settings__ > __Integrations__ > __Servers & Services__.
```

Comment:
detailed description should contain only the necessary info to config the instance.
how to create API Key, permissions required, etc.
not the whole documentation generated. This documentation should be attached to new Documentation issue